test/cpp/interop/interop_server.cc

```diff
@@ -333,6 +340,10 @@ void grpc::testing::interop::RunServer(
   }
   std::unique_ptr<Server> server(builder.BuildAndStart());
   gpr_log(GPR_INFO, "Server listening on %s", server_address.str().c_str());
+
+  // Signal that the server has started.
+  if (server_started_condition) server_started_condition->notify_all();
```

Comment:
where's the wait on the condvar?

Reply:
The wait on the condvar would be in the calling code. This allows a client to start the server in a new / separate thread and then wait on the condvar until the server was started.

Reply:
The client may get falsely notified from a spurious wakeup. Not sure whether that matters for you though.

Reply:
Thanks for bringing this up! Added code to allow clients to correctly detect & handle a spurious wakeup.

Reply:
LGTM